snaps/learn/tutorials/gas-estimation.md

```diff
@@ -199,10 +242,10 @@ case "hello":
 
 To build and test your Snap:
 
-1. Open `package.json` in the root directory of the project, and increment the `"version"` (if the `"version"` is
-   `0.1.0`, increase it to `0.2.0`).
+5.1 (Optional for testing) open `package.json` in the root directory of the project, and increment 
+  the `"version"` (if the `"version"` is `0.1.0`, increase it to `0.2.0`).
```

Comment:
```suggestion
#### 5.1. (Optional) Increment the version

Open `package.json` in the root directory of the project, and increment 
  the `"version"` (if the `"version"` is `0.1.0`, increase it to `0.2.0`).
  This is optional for testing.
```

If we want to remove the ordered list and use sub-steps like "5.1," I suggest adding these as H4s for visibility and consistency.

Reply:
yes, I like it better, but not convinced about H4s going into toc --> makes the tutorial seem more imposing